apps/geo/serializers.py

```diff
@@ -47,7 +47,7 @@ class RegionSerializer(RemoveNullFieldsMixin,
         write_only=True,
         required=False,
     )
-
+    is_published = serializers.BooleanField(read_only=True)
```

Comment:
Flake8.  :expressionless: 

```suggestion
    is_published = serializers.BooleanField(read_only=True)

```